examples/2d/text2d.rs

```diff
@@ -132,44 +132,46 @@ fn setup(mut commands: Commands, asset_server: Res<AssetServer>) {
         Text2dShadow::default(),
     ));
 
-    commands
-        .spawn((
-            Sprite {
-                color: Color::Srgba(LIGHT_CYAN),
-                custom_size: Some(Vec2::new(10., 10.)),
-                ..Default::default()
-            },
-            Transform::from_translation(250. * Vec3::Y),
-        ))
-        .with_children(|commands| {
-            for (text_anchor, color) in [
+    commands.spawn((
+        Sprite {
+            color: Color::Srgba(LIGHT_CYAN),
+            custom_size: Some(Vec2::new(10., 10.)),
+            ..Default::default()
+        },
+        Transform::from_translation(250. * Vec3::Y),
+        Children::spawn(SpawnIter(
```

Comment:
not a fan of this one, but won't block on it

Reply:
I don't think I am either but I also didn't know about this functionality so maybe it's worth it!

Reply:
The API in this case just looks clunky to me, no matter what. But that might just be me being more used to the "old" way :)